buffers-and-streams/buffers.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Buffers in depth
+url: buffers
+author: john
+date: 2019-11-14
+---
+
+## Buffers
+
+Node's `Buffer` class enables working with binary data in Javascript. A buffer allows a programme to store data temporarily outside the V8 engine's stack.
+
+Though not entirely accurate we can think of a buffer as an array of bytes, each byte is represented by a hexadecimal digit.
+
+The Node `Buffer` class is a global class and therefore does not need to be imported into a module.
+
```

Comment:
what are buffer's used for - a real world example

answer the `so what?` question